.github/workflows/e2e.yml

```diff
@@ -9,7 +9,7 @@ concurrency:
 
 jobs:
   e2e:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why is this necessary? Latest is too new for Cypress? Or too old?

Reply:
Our Cypress tests started to fail due to using latest runner image which conflicts with Cypress version. -latest tag uses image which might not be that stable as previous one, or be in pre-release mode.

Reply:
By specifying a version, we ensure that there will be some control over it. When this version is planned to become unavailable, we will hopefully receive notification that will give us enough time for updating to newer version of Cypress/resolve any conflicts instead of reacting to this when it was unexpected.